source/config/configSpec.py

```diff
@@ -126,6 +126,10 @@
 
 [brailleViewer]
 	showBrailleViewerAtStartup = boolean(default=false)
+	shouldHoverRouteToCell = boolean(default=false)
+	secondsOfHoverToActivate = float(min=0.0, default=1.0)
+	# Devices with 40 cells are quite common.
+	defaultCellCount = integer(min=20, max=160, default=40)

```

Comment:
There are some displays smaller than 20 cells. Not enforcing the lower limit would make the implementation more flexible IMO.

Reply:
I agree. There are displays with 12 cells. And even some prototype displays in a rolling wheel shape with only 1 cell. So I think we should ensure this works for smaller than 20.

Reply:
Note, this config is not exposed as a user setting by this PR. I think these considerations should be made when that happens, these values were just chosen as conservative values (to keep testing to minimum, and common utility high).

I'd prefer to follow this PR up with with one to make this configurable, and look more closely at the range that we should support. However, if a physical braille display is connected the braille viewer will adjust to match the number of cells

Realistically, the braille viewer would have very limited utility with only one cell. One cell relies on the ease of scrolling that comes with rolling the wheel. Further I'm not sure I understand the use case of trying to mimic this with the braille viewer.